2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -127,6 +127,7 @@ package model {
   final case class CollectorConfig(
     interface: String,
     port: Int,
+    paths: Option[Map[String, String]],
```

Comment:
I think we'd want to make it mandatory as options don't play well with configuration templating

Reply:
What is users don't want to map anything? Would we require them to pass in the config:

```
paths {
  "r/tp2" = "r/tp2"
}
```

?

Reply:
if they don't want to map anything they would specify `paths { }`

Reply:
Ah, and then that would be parsed as `Map.empty`?

Reply:
Done.